server/shadow/shadow_server.c

```diff
@@ -220,11 +225,31 @@ int shadow_server_parse_command_line(rdpShadowServer* server, int argc, char** a
 		}
 		CommandLineSwitchCase(arg, "ipc-socket")
 		{
+			/* /bind-address is incompatible */
+			if (server->ipcSocket)
+				return -1;
+
 			server->ipcSocket = _strdup(arg->Value);
 
 			if (!server->ipcSocket)
 				return -1;
 		}
+		CommandLineSwitchCase(arg, "bind-address")
+		{
+			int rc;
+			size_t len = strlen(arg->Value) + sizeof(bind_address);
+			/* /ipc-socket is incompatible */
```

Comment:
The usage message doesn't make this clear, I think a specific error message would be nice to have here